app/src/main/res/values/strings.xml

```diff
@@ -126,6 +126,8 @@
     <string name="prohibitedAction">Check that you have necessary permissions</string>
     <string name="editXWikiContactInfo">Edit XWiki contact info</string>
     <string name="unchangedSettings">Nothing to save: settings was not changed</string>
+    <string name="ok">OK</string>
```

Comment:
There is standard [ok](https://developer.android.com/reference/android/R.string#ok) string in Android